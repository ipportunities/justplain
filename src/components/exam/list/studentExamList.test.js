import React from 'react'
import { shallow } from 'enzyme'
import { StudentExamList } from './index'

jest.mock('react-redux', () => ({
  ...jest.requireActual('react-redux'),
  useSelector: jest.fn().mockReturnValue({ auth: { token: 'a' } })
}))

const examTypeMock = {}
const setExamStartDataMock = jest.fn()

describe('Student Exam List', () => {
  it('should render correctly', () => {
    const component = shallow(<StudentExamList examType={ examTypeMock } setExamStartData={ setExamStartDataMock } />)
    expect(component).toMatchSnapshot()
  })
})
