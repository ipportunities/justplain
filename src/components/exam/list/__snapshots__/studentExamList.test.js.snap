// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Student Exam List should render correctly 1`] = `<div />`;
